python/mleap/pyspark/feature/string_map.py

```diff
@@ -53,7 +92,17 @@ def from_dataframe(cls, labels_df, inputCol, outputCol, handleInvalid='error', d
         See StringMap() for other params.
         """
         assert isinstance(labels_df, DataFrame), 'labels must be a DataFrame, got: {}'.format(type(labels_df))
-        labels_dict = {row[0]: float(row[1]) for row in
-                       labels_df.select([inputCol, outputCol]).collect()}
-        return cls(labels_dict, inputCol=inputCol, outputCol=outputCol, handleInvalid=handleInvalid,
-                   defaultValue=defaultValue)
+        labels_dict = {
+            row[0]: float(row[1])
+            for row in
```

Comment:
nitpick, is the extra space required?

Reply:
I think the space is only in the github UI, there's no newline between line 97 and 98, or is that what you meant ?